### PR TITLE
Update microsoft-365-vpn-implement-split-tunnel.md

### PR DESCRIPTION
**Pull request by msbemba**
Added a question Faq on Teams support scenario with browser and desktop client.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/2784

**Comment by msbemba** [comment]
@JohanFreelancer9  Please copy review

File: microsoft-365/enterprise/microsoft-365-vpn-implement-split-tunnel.md [hunk 1]
```diff
@@ -324,6 +324,10 @@ Port 80 is only used for things like redirect to a port 443 session, no customer
 
 **No**, it does not. The one caveat to the above advice is users in the PRC who are connecting to a worldwide instance of Office 365. Due to the common occurrence of cross border network congestion in the region, direct Internet egress performance can be variable. Most customers in the region operate using a VPN to bring the traffic into the corporate network and utilize their authorized MPLS circuit or similar to egress outside the country via an optimized path. This is outlined further in the article [Office 365 performance optimization for China users](microsoft-365-networking-china.md).
 
+### Does split-tunnel configuration  works for  Teams running in a browser?
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/microsoft-365-vpn-implement-split-tunnel.md]:
```suggestion
### Does split-tunnel configuration work for Teams running in a browser?
```

File: microsoft-365/enterprise/microsoft-365-vpn-implement-split-tunnel.md [hunk 2]
```diff
@@ -324,6 +324,10 @@ Port 80 is only used for things like redirect to a port 443 session, no customer
...
+
+**No**, It does not. It works only on  Microsoft Teams client version 1.3.00.13565 or greater. This version includes improvements in how the client detects available network paths.
```
Comment (JohanFreelancer9) [inline microsoft-365/enterprise/microsoft-365-vpn-implement-split-tunnel.md]:
```suggestion
**No**, it does not. It works only on Microsoft Teams client version 1.3.00.13565 or greater. This version includes improvements in how the client detects available network paths.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kelleyvice-msft Please merge this PR. Thanks.

**Pull request merged**